Lib/_pyrepl/utils.py

```diff
@@ -2,12 +2,56 @@
 import unicodedata
 import functools
 
+from idlelib import colorizer
```

Comment:
```suggestion
from idlelib import colorizer
```

On Windows, if you do not choose to install Tcl/Tk, IDLE will not be available

See https://github.com/python/cpython/blob/main/Tools%2Fmsi%2Ftcltk%2Ftcltk.wixproj